.ci/travis-before-script.sh

```diff
@@ -4,9 +4,24 @@ set -e -x
 
 if [[ "${TOXENV}" == *-rdb ]]; then
     rethinkdb --daemon
-elif [[ "${BIGCHAINDB_DATABASE_BACKEND}" == mongodb ]]; then
-    wget http://downloads.mongodb.org/linux/mongodb-linux-x86_64-3.4.1.tgz -O /tmp/mongodb.tgz
+elif [[ "${BIGCHAINDB_DATABASE_BACKEND}" == mongodb && \
+        -z "${BIGCHAINDB_DATABASE_SSL}" ]]; then
```

Comment:
This is becoming bigger and a bit more complex. Lets put in some comments explaining the separation of these two cases.